From: Tiezhu Yang <yangtiezhu@loongson.cn>
Subject: docs: submit-checklist: remove all of "make namespacecheck"
Date: Tue, 19 Dec 2023 20:50:01 +0800

After commit 7dfbea4c468c ("scripts: remove namespace.pl"),
scripts/namespace.pl has been removed from the kernel, and "make
namespacecheck" has been removed from the English version of
submit-checklist.rst, so also remove it in the related translations.

Link: https://lkml.kernel.org/r/20231219125008.23007-6-yangtiezhu@loongson.cn
Signed-off-by: Tiezhu Yang <yangtiezhu@loongson.cn>
Cc: Jonathan Corbet <corbet@lwn.net>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/translations/ja_JP/SubmitChecklist              |    4 ++--
 Documentation/translations/zh_CN/process/submit-checklist.rst |    3 +--
 Documentation/translations/zh_TW/process/submit-checklist.rst |    3 +--
 3 files changed, 4 insertions(+), 6 deletions(-)

--- a/Documentation/translations/ja_JP/SubmitChecklist~docs-submit-checklist-remove-all-of-make-namespacecheck
+++ a/Documentation/translations/ja_JP/SubmitChecklist
@@ -56,8 +56,8 @@ などのLinuxカーネルパッチ投�
 
  9: sparseを利用してちゃんとしたコードチェックをしてください。
 
-10: 'make checkstack' と 'make namespacecheck' を利用し、問題が発見されたら
-    修正してください。'make checkstack' は明示的に問題を示しませんが、どれか
+10: 'make checkstack' を利用し、問題が発見されたら修正してください。
+    'make checkstack' は明示的に問題を示しませんが、どれか
     １つの関数が512バイトより大きいスタックを使っていれば、修正すべき候補と
     なります。
 
--- a/Documentation/translations/zh_CN/process/submit-checklist.rst~docs-submit-checklist-remove-all-of-make-namespacecheck
+++ a/Documentation/translations/zh_CN/process/submit-checklist.rst
@@ -53,8 +53,7 @@ 和其他有关提交Linux内核补丁�
 9) 通过 sparse 清查。
    （参见 Documentation/translations/zh_CN/dev-tools/sparse.rst ）
 
-10) 使用 ``make checkstack`` 和 ``make namespacecheck`` 并修复他们发现的任何
-    问题。
+10) 使用 ``make checkstack`` 并修复他们发现的任何问题。
 
     .. note::
 
--- a/Documentation/translations/zh_TW/process/submit-checklist.rst~docs-submit-checklist-remove-all-of-make-namespacecheck
+++ a/Documentation/translations/zh_TW/process/submit-checklist.rst
@@ -56,8 +56,7 @@ 和其他有關提交Linux內核補丁�
 9) 通過 sparse 清查。
    （參見 Documentation/translations/zh_CN/dev-tools/sparse.rst ）
 
-10) 使用 ``make checkstack`` 和 ``make namespacecheck`` 並修復他們發現的任何
-    問題。
+10) 使用 ``make checkstack`` 並修復他們發現的任何問題。
 
     .. note::
 
_
